internal/pkg/testhelpers/iptbtester/genesis.go

```diff
@@ -27,7 +28,7 @@ type idResult struct {
 
 // MustGenerateGenesis constructs the requires information and files to build a single
 // filecoin node with the provided funds. The GenesisInfo can be used with MustImportGenesisMiner
-func MustGenerateGenesis(t *testing.T, funds int64, dir string) *GenesisInfo {
+func MustGenerateGenesis(t *testing.T, funds int64, dir string, genesisTime time.Time) *GenesisInfo {
```

Comment:
another old nit: since you are touching all of these calls consider renaming the function to `RequireGenerateGenesis` to follow our convention: require means exit with t.Fatal in case of error, must means panic in case of error.

Reply:
done